### PR TITLE
week 4 mapAPI

### PR DESCRIPTION
**Pull request by honeybeebaby**
added map API to page

### CODE REVIEW

**Comment by ccondit** [comment]
Michaela and Ihsan - want to take a first look here and then hand off to Kaveh and I for final LGTM?

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -96,5 +97,8 @@ <h1>a page about me</h1>
             </div>
         </div>
 
+        <div id="map">
+        </div>
+
```
Comment (mmurray22) [inline portfolio/src/main/webapp/index.html]: nit-remove extra space after div

**Review by mmurray22 — APPROVED** [review]
LGTM! Only one nit :)

**Review by ccondit — APPROVED** [review]
I'll follow up next week with a formatting solution for html / javascript. This LGTM for now.

**Review by kaveh096 — APPROVED** [review]

**Review by ihsan314 — APPROVED** [review]
LGTM

**Pull request merged**